gw2rpc/settings.py

```diff
@@ -8,11 +8,13 @@ def __init__(self):
         if not os.path.exists("config.ini"):
             config["API"] = {"APIKey": ""}
             config["Discord"] = {"UpdateFrequency": 10}
+            config["CloseWithGw2"] = {"CloseWithGw2": True}
             with open("config.ini", "w") as cfile:
                 config.write(cfile)
         config.read("config.ini")
         self.api_key = config["API"]["APIKey"]
         self.update_frequency = config.getint("Discord", "UpdateFrequency")
+        self.close_with_gw2 = config.getboolean("CloseWithGw2", "CloseWithGw2")
```

Comment:
Will raise an unhandled exception in case of missing value/invalid value. Needs a fallback value as well.